airflow/providers/jira/sensors/jira.py

```diff
@@ -98,10 +99,9 @@ def __init__(self,
 
         super().__init__(jira_conn_id=jira_conn_id,
                          result_processor=field_checker_func,
-                         *args,
```

Comment:
Remove *args to avoid mypy issue:
https://github.com/python/mypy/issues/6799
https://github.com/apache/airflow/pull/9784

Reply:
+ https://github.com/apache/airflow/issues/9942

Reply:
Can you help me with it? Can you help me with that too? That way, other people won't face the problem you had.

Reply:
Sure will look into soon.